siswrap/handlers.py

```diff
@@ -124,12 +155,12 @@ def post(self, runfolder="/some/runfolder"):
                     "runfolder": result.info.runfolder,
                     "link": result.info.link,
                     "msg": result.info.msg,
-                    "service_version": my_version,
-                    "sisyphus_version": sisyphus_version}
+                    "service_version": siswrap_version,
+                    "sisyphus_version": wrapper.sisyphus_version()}
+
             self.write_accepted(resp)
         except RuntimeError, err:
-            self.write_object("An error ocurred: " + str(err),
-                              http_code=500, reason="An error occurred")
+            raise tornado.web.HTTPError(500, "An error occurred: {0}".format(str(err)))
```

Comment:
Yeah, exactly. I noticed that Steinar is using this method for 'runfolder', and it helped me quickly work around the list as output issue.